pkg/generated/clientset/versioned/clientset.go

```diff
@@ -28,8 +28,6 @@ import (
 type Interface interface {
```

Comment:
not sure what's this commit, can we just have a commit for godep changes

Reply:
Resolved

Reply:
can you please explain why we need this commit?